src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -31,13 +31,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String user = request.getParameter("user");
     if (user == null || user.equals("")) {
       System.err.println("Error from AboutMeServlet: User parameter invalid.");
-      response.sendRedirect("/index.html");
+      response.sendRedirect("/");
       return;
     }
 
     User userData = datastore.getUser(user);
-    if (userData == null || userData.getAboutMe() == null) {
-      response.getOutputStream().println("This \"About me\" page is empty :(");
+    if (userData == null) {
+      System.err.println("Error from AboutMeServlet: User does not exist");
+      response.sendRedirect("/invalid-user/" + user);
       return;
```

Comment:
This, for example, is fine to go away  as long as by time we get to here, we know user is valid. (say, if we check in user.jsp)